js/script.js

```diff
@@ -1186,11 +1186,18 @@ Array.prototype.unique = function() {
 		favArray : [],
 		add : function(){
 			var latlng = $(this).attr("data-latlng").split(",");
-			var formData = {
-				lat : latlng[0],
-				lng : latlng[1]
-			};
-			$.post(OC.generateUrl('/apps/maps/api/1.0/favorite/addToFavorites'), formData);
+			OC.dialogs.prompt("Enter a name", "Name your favorite", function(isYes, name){
```

Comment:
Yep. Should replace the name of the place, and use the name of the place as the default favorite name. Like when a place is called »Café Lost Weekend«, the favorite should also by default have that label.